OvaryModelGenerator/CMakeLists.txt

```diff
@@ -47,8 +47,8 @@ target_link_libraries(TestingHelper PRIVATE ${VTK_LIBRARIES})
 
 # target_link_libraries(Main PRIVATE ${VTK_LIBRARIES})
 add_library(OvaryModelGenerator OvaryModelGenerator.cxx)
-add_executable(Main Main.cxx PelvisModelGenerator.cxx)
-target_link_libraries(Main PRIVATE ${VTK_LIBRARIES} OvaryModelGenerator)
+add_executable(ModelGenerator Main.cxx PelvisModelGenerator.cxx)
```

Comment:
Main is normally not used for binary name. Because user calling it should know some thing more specific, although brief, about what this binary is about. Changing to ModelGenerator here to reflect our planning to adding more types of models into this program.